src/sulogin-shell/sulogin-shell.c

```diff
@@ -89,15 +90,28 @@ static void print_mode(const char* mode) {
 }
 
 int main(int argc, char *argv[]) {
-        static const char* const sulogin_cmdline[] = {SULOGIN, NULL};
+        /* Note: sulogin_cmdline double-NULL is intentional!
+         * (The first null might be overwritten with --force argument)
+         */
+        static const char* sulogin_cmdline[] = {SULOGIN, NULL, NULL};
```

Comment:
hmm, in other cases like this we added line breaks and commented individually what the entries are for. i.e.:

```c
static const char* sulogin_cmdline[] = {
        SULOGIN, 
        NULL,             /* --force */
        NULL
};
```

i think this is a nicer way to explain this, and it's good to follow the same schemes throughout the codebase

Reply:
oh, and also, if we modify this, then this should not be "static"

Reply:
thanks for suggestions, both should be adressed in updated version.